SettingsList.py

```diff
@@ -2249,19 +2249,35 @@ def __init__(self, name, gui_text, min, max, default, step=1,
         gui_params     = {
             'randomize_key': 'randomize_settings',
         },
-    ), 
-    Checkbutton(
+    ),
+    Combobox(
         name           = 'mix_entrance_pools',
         gui_text       = 'Mix Entrance Pools',
+        default        = 'off',
+        choices        = {
+            'off':       'Off',
+            'non-ow':    'Non-Overworld',
+            'all':       'All Entrances',
+        },
         gui_tooltip    = '''\
+            'Non-Overworld':
+            Shuffle all interior entrances into a single mixed pool.
+            This means any shuffled entrance could lead to any type
+            of area: Dungeon, Grotto, or Interior. Overworld
+            entrances are kept in a separate pool if randomized.
+    
+            'All Entrances':
             Shuffle all entrances into a single mixed pool.
             This means any shuffled entrance could lead to any type 
             of area: Dungeon, Grotto, Interior or Overworld.
```

Comment:
I think we should reword the description to something along these lines, for clarity and to avoid duplicating text. Let me know what you think.
```suggestion
            Shuffle entrances into mixed pools instead of separate ones.

            'Indoor Entrances':
            Mixes "Indoor" entrance pools, which correspond to entrances
            of type Dungeon, Grotto or Interior. Overworld entrances
            are kept in their own separate pool if randomized.
    
            'All Entrances':
            Mixes all entrance pools, including both "Indoor" and
            Overworld entrances.
```

Reply:
Should "Shuffle entrances into mixed pools" be "Shuffle entrances into a mixed pool"?

Reply:
I think this option might end up being more than just one mixed pool in the future pool, so this more general wording is probably better. I think both would be fine right now tho.

Reply:
With how it's currently implemented I think it's better to stay singular.

Reply:
Yeah that's probably fine